src/NzbDrone.Core.Test/ParserTests/UnicodeReleaseParserFixture.cs

```diff
@@ -47,7 +47,35 @@ public void should_parse_chinese_anime_releases(string postTitle, string title,
         [TestCase("[SweetSub&LoliHouse] 来自深渊 烈日黄金乡 / Anime-Series Title S2 - 07 [WebRip 1080p HEVC-10bit AAC][简繁日内封字幕]", "Anime-Series Title", "SweetSub&LoliHouse", 2, 7)]
         [TestCase("[LoliHouse] Love Live! 虹咲学园学园偶像同好会 第二季 / Anime-Series Title S2 - 10 [WebRip 1080p HEVC-10bit AAC][简繁内封字幕]", "Anime-Series Title", "LoliHouse", 2, 10)]
         [TestCase("[澄空学园&雪飘工作室&LoliHouse] 辉夜大小姐想让我告白 第三季 / Anime-Series Title S3 - 06 [WebRip 1080p HEVC-10bit AAC][简繁内封字幕]", "Anime-Series Title", "澄空学园&雪飘工作室&LoliHouse", 3, 6)]
-        public void should_parse_chinese_anime_season_episode_releases(string postTitle, string title, string subgroup, int seasonNumber, int episodeNumber)
+        public void should_parse_chinese_anime_season_episode_releases_1(string postTitle, string title, string subgroup, int seasonNumber, int episodeNumber)
```

Comment:
These can all be in one fixture. `should_parse_chinese_anime_season_number_with_episode_number_releases`